### PR TITLE
removed a channel

### PR DESCRIPTION
**Pull request by satdav**
removed the contributors channel as now been redirected to Mozillians

### CODE REVIEW

**Comment by whimboo** [comment]
This full patch is not a change we want for the testdaybot in the example config. You can easily update the config file according to your needs, and then start the bot on your server. So please kill all the changes beside the #contributors channel replacement. That one I'm happy to push.

**Comment by satdav** [comment]
@whimboo  its ready now when you are

File: config-example.json [hunk 1]
```diff
@@ -17,14 +17,14 @@
...
     "advertisement":{
         "channels":[
-            "#contributors",
             "#developers",
             "#interns",
             "#introduction",
             "#mozillians",
             "#newbies",
             "#seneca"
+
```
Comment (whimboo) [inline config-example.json]: I added an empty line here. You can simply remove it.

File: config-example.json [hunk 2]
```diff
@@ -17,14 +17,14 @@
...
         ],
-        "message":"Mozilla QA is holding a Test Day today. Interested in participating? Please join us in #qa!"
+        "message":"Mozilla QA is holding a Test Day today. Interested in participating? Please join us in #qa !"
```
Comment (whimboo) [inline config-example.json]: Please also revert this change.

**Comment by davehunt** [comment]
Closing as this has not been active for over three years and likely not a change we need any more.
